src/fastnetmon.cpp

```diff
@@ -1015,25 +1015,85 @@ void parse_hostgroups(std::string name, std::string value) {
         << host_groups[ host_group_name ].size() << " subnets";
 }
 
+const char * const CONFIG_DIRECTIVE_INCLUDE_FILE = "include_file";
+
+bool configuration_is_include_file_directive(const std::string & s) {
+    return (s.find(CONFIG_DIRECTIVE_INCLUDE_FILE) == 0);
+}
+
 // Load configuration
 bool load_configuration_file() {
-    std::ifstream config_file(global_config_path.c_str());
-    std::string line;
+    if (file_exists(global_config_path) == false) {
+        logger << log4cpp::Priority::ERROR << "Config file does not exist";
+        return false;
+    }
 
-    if (!config_file.is_open()) {
-        logger << log4cpp::Priority::ERROR << "Can't open config file";
+    std::vector<std::string> lines_from_config = read_file_to_vector(global_config_path);
+    if(lines_from_config.size() == 0) {
+        logger << log4cpp::Priority::ERROR << "Config file is empty";
         return false;
     }
 
-    while (getline(config_file, line)) {
+    // process derectives:
+    //   - 'include path_to_file_to_be_included' - replace this item with items from file
+    // In this loop we have to break if nothing found and return from function to prevent dead loop!
+    while(true) {
+        // find line which may be 'include' directive
+        std::vector<std::string>::iterator fit = std::find_if(
```

Comment:
Clean solution here is very complicated but it's OK to fill new data to new array instead of in-place vector changes.